packages/nimble-components/src/table-column/enum-text/index.ts

```diff
@@ -0,0 +1,85 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { TableColumnEnumBase, TableColumnEnumColumnConfig } from '../enum-base';
+import { styles } from '../enum-base/styles';
+import { template } from '../enum-base/template';
+import { TableColumnSortOperation, TableColumnValidity } from '../base/types';
+import { mixinGroupableColumnAPI } from '../mixins/groupable-column';
+import { mixinFractionalWidthColumnAPI } from '../mixins/fractional-width-column';
+import { MappingText } from '../../mapping/text';
+import { TableColumnEnumTextValidator } from './models/table-column-enum-text-validator';
+import type { ColumnInternalsOptions } from '../base/models/column-internals';
+import { tableColumnEnumTextCellViewTag } from './cell-view';
+import { tableColumnEnumTextGroupHeaderViewTag } from './group-header-view';
+import type { Mapping } from '../../mapping/base';
+import type { MappingConfig } from '../enum-base/models/mapping-config';
+import { MappingTextConfig } from '../enum-base/models/mapping-text-config';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-enum-text': TableColumnEnumText;
+    }
+}
+
+/**
+ * Table column that maps values to strings
+ */
+export class TableColumnEnumText extends mixinGroupableColumnAPI(
+    mixinFractionalWidthColumnAPI(
+        TableColumnEnumBase<
+        TableColumnEnumColumnConfig,
+        TableColumnEnumTextValidator
+        >
+    )
+) {
+    protected get supportedMappingElements(): readonly (typeof Mapping)[] {
```

Comment:
I'm wondering if it's really important for this member to be defined as an abstract member. Its only usage is to be passed as a constructor parameter, so it doesn't really have utility as a `protected` member (useful to derivations). The `createValidator()` implementation seems like it could easily just be:
```ts
    public override createValidator(): TableColumnEnumTextValidator {
        return new TableColumnEnumTextValidator(
            this.columnInternals,
            [MappingText]
        );
    }
```